pom.xml

```diff
@@ -563,73 +563,73 @@
         <dependency>
             <groupId>org.eclipse.rdf4j</groupId>
             <artifactId>rdf4j-rio-binary</artifactId>
-            <version>${rdf4j-rio.version}</version>
+            <version>3.0.4</version>
```

Comment:
I think we can create a custom variable with 3.0.4 version number

Reply:
Oops, this is a bad merge - we already have that in dev. Will fix.